x-pack/plugins/threat_intelligence/public/modules/block_list/hooks/use_policies.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useQuery } from '@tanstack/react-query';
+import { useKibana } from '../../../hooks';
+
+const POLICIES_URL = '/api/fleet/package_policies';
+const PACKAGE_POLICY_SAVED_OBJECT_TYPE = 'ingest-package-policies';
+
+export interface PolicyResponse {
+  items: Policy[];
+}
+
+export interface Policy {
+  id: string;
+  name: string;
+}
+
+export function usePolicies() {
+  const { http } = useKibana().services;
+  const queryKey = ['policies'];
+
+  const fetchPolicies = () =>
```

Comment:
would it make sense to `useCallback` here?

Reply:
I honestly didn't think about it, as we haven't used useCallback in the other places we've used useQuery...